packages/web-components/fast-components/src/listbox/listbox.styles.ts

```diff
@@ -38,43 +65,56 @@ export const listboxStyles: FoundationElementTemplate<ElementStyles> = (
             padding: calc(${designUnit} * 1px) 0;
         }
 
-        :host(:focus-within:not([disabled])) {
-            border-color: ${focusStrokeOuter};
-            box-shadow: 0 0 0 calc((${focusStrokeWidth} - ${strokeWidth}) * 1px)
-                ${focusStrokeOuter} inset;
-        }
+        ${!hostContext ? css`
+            :host(:focus-within:not([disabled])) {
+                border-color: ${focusStrokeOuter};
+                box-shadow: 0 0 0
+                    calc((${focusStrokeWidth} - ${strokeWidth}) * 1px)
+                    ${focusStrokeOuter} inset;
+            }
+
+            :host([disabled]) ::slotted(*) {
+                cursor: ${disabledCursor};
+                opacity: ${disabledOpacity};
+                pointer-events: none;
+            }
+        ` : ``}
 
-        :host([disabled]) ::slotted(*) {
-            opacity: ${disabledOpacity};
-            cursor: ${disabledCursor};
-            pointer-events: none;
+        ::slotted([aria-selected="true"][aria-checked="false"]) {
+            border-color: ${neutralLayerFloating};
```

Comment:
If this is intended to be the same color as the background, can it be "transparent" instead. It's confusing to see a "layer" recipe used as a border.